### PR TITLE
Changing the limit for JSON bodyParser (#372)

### PR DESCRIPTION
**Pull request by guyemerson**
Add sizeLimit as an option, which controls the maximum size of a JSON body that can can be accepted by bodyParser.json.  This option is available both at the command line and as middleware.

### CODE REVIEW

**Comment by guyemerson** [comment]
The only check that is still failing is to do with the version of graphql, but that is not affected by this pull request.

File: docs/cli.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ The usage of the `postgraphql` binary is as follows. To pull up this documentati
     -a, --classic-ids                use classic global id field name. required to support Relay 1
     -j, --dynamic-json               enable dynamic JSON in GraphQL inputs and outputs. uses stringified JSON by default
     -M, --disable-default-mutations  disable default mutations, mutation will only be possible through Postgres functions
+    -l, --size-limit                 set the maximum size of JSON bodies that can be parsed (default 100kB)
```
Comment (calebmer) [inline docs/cli.md]: Could we change the name to `--body-size-limit`? Could we also add a note that you can define it using a pretty size like `200kB`.

File: docs/library.md [hunk 2]
```diff
@@ -64,6 +64,7 @@ Arguments include:
   - `enableCors`: Enables some generous CORS settings for the GraphQL endpoint. There are some costs associated when enabling this, if at all possible try to put your API behind a reverse proxy.
   - `exportJsonSchemaPath`: Enables saving the detected schema, in JSON format, to the given location. The directories must exist already, if the file exists it will be overwritten.
   - `exportGqlSchemaPath`: Enables saving the detected schema, in GraphQL schema format, to the given location. The directories must exist already, if the file exists it will be overwritten.
+  - `sizeLimit`: Set the maximum size of JSON bodies that can be parsed (default 100kB).
```
Comment (calebmer) [inline docs/library.md]: Could we change this to `bodySizeLimit` and add a similar note about pretty bytes?

**Comment by calebmer** [comment]
I tried restarting the build. I’ll investigate if it keeps failing 😣

**Comment by benjie** [comment]
Thanks! 👍

**Pull request merged**